### PR TITLE
everything works

### PR DESCRIPTION
**Pull request by gyrydymptum**
ready function to be checked

### CODE REVIEW

**Comment by ziobron** [comment]
⏱ 3 XP extra for @gyrydymptum for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,29 @@
...
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    int resultint;
+    std::string result;
+        if (command == "add"){
+            resultint = first + second;
+            result = std::to_string(resultint);
+            }
```
Comment (ziobron) [inline homework/calculate/calculate.hpp:10]: The code can be simplified in this way:
```suggestion
        if (command == "add"){
            return std::to_string(first + second);
        }
```

**Review by ziobron — APPROVED** [review]
there are some minor formatting issues.

**Review by gyrydymptum — COMMENTED** [review]
test

**Pull request closed without merging**